framework/source/class/qx/ui/list/List.js

```diff
@@ -597,6 +625,28 @@ qx.Class.define("qx.ui.list.List",
     },
 
 
+    /**
+     * Event handler for the updated event of the 
+     * qx.ui.virtual.layer.WidgetCell layer.
+     *
+     * Recalculates the item sizes in a deffered call,
+     * which only happens if we have variable item heights
+     */
+    _onLayerUpdated: function () {
+      if (this.isVariableItemHeight() === false) {
+        return;
+      }
+
+      if (this.__deferredLayerUpdate === null) {
+        this.__deferredLayerUpdate = new qx.util.DeferredCall(function () {
+          this._setRowItemSize();
+        }, this);
+      }
+
+      this.__deferredLayerUpdate.schedule();
+    },
+
```

Comment:
@derrell oops, sorry crossed in the post there.  The `schedule` needs to stay where it is because it needs to be called each time there is an update